CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.0.4
+ADDED commando.commands.builtin/commando-macro-spec. The new type of command that allow to group instructions by its functionality and use it as a single command. Added Readme information about the macro.
+ADDED documentation for commando.commands.builtin commands. Now each built-in command have explanation of its behavior and examples of usage.
+UPDATED upgrade commando.commands.query-dsl. Function `resolve-query` was removed and replaced by `resolve-fn`, `resolve-instruction`, `resolve-instructions-qe` function called a **resolvers**. Explanations about the resolvers added to _docs/query-dsl.md_ file.
+UPDATED error serialization. `commando.impl.utils` contains new way to serialize errors for CLJ/CLJS. Now all errors are serialized to map with keys: `:type`, `:class`, `:message`, `:data` (if exists) and `:stacktrace` (if exists), `:cause` (if exists). See `commando.impl.utils/serialize-error` for more information. You can expand the error handlers using `serialize-exception-fn` multimethod (but for CLJ only).
+ADDED tests for macro-spec, errors and query-dsl changes.
+UPDATED README.md 'Debugging section' was replaced on 'Configuring Execution Behavior' which contains more detailed information how to modify execution behavior.
+UPDATED dynamic variable *debug-mode* replaced by the `*execute-config*` which is a map that can contain multiple configuration options.
+
```

Comment:
I would add here info about:
 FIXED: Removed `detach-instruction-commands` call from build-compiler**
   - In `commando.core/build-compiler`, the line that detached instruction commands from the registry was removed
   - This means the compiled registry now **includes** internal commands (`_map`, `_value`, `_vector`)